src/auth-service/utils/create-user.js

```diff
@@ -468,40 +455,13 @@ const join = {
                 : "",
             };
           } else if (responseFromSendEmail.success === false) {
-            return {
-              success: false,
-              message: responseFromSendEmail.message,
-              errors: responseFromSendEmail.errors
-                ? responseFromSendEmail.errors
-                : "",
-              status: responseFromSendEmail.status
-                ? responseFromSendEmail.status
-                : "",
-            };
+            return responseFromSendEmail;
           }
         } else if (responseFromSaveToken.success === false) {
-          return {
-            success: false,
-            message: responseFromSaveToken.message,
```

Comment:
just simple refactoring to reduce on lines of code and also align all the error responses to the AirQo standard of JSON response =`{errors: {message:"error message"}}`